src/views/List.jsx

```diff
@@ -12,7 +11,12 @@ export function List({ data, listPath, lists }) {
 	const [searchItem, setSearchItem] = useState('');
 	const [items, setItems] = useState([]);
 
-	const listTitle = listPath.split('/')[1];
+	const listTitle = listPath ? listPath.split('/')[1] : null;
+	const fixedListTitle = listTitle
```

Comment:
Just curious about why are you a you adding this fix to the list title 🤔

Reply:
When we create a new list we are automatically redirected to the `List` view. I noticed that after the redirect, the list title would render with the symbol '}' at the end of the string for some reason. It would disappear after navigating to `Home` view and then clicking on the new list button again